tests/test_models.py

```diff
@@ -0,0 +1,30 @@
+"""Test that all aiida entity models can be loaded loaded into pydantic models."""
+from aiida_restapi import models
+from aiida import orm
+
+
+def replace_dynamic(data: dict) -> dict:
```

Comment:
Just to understand: are you doing this just here for testing purposes (which is certainly fine) or are you proposing that this should e.g. become the response of the API (which would require some thought).

Reply:
oh yeh just for testing purposes; to replace things that are not reproducible

Reply:
could you mention this in the doc string?